apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -568,7 +568,8 @@ const Ticket = new GQLListSchema('Ticket', {
             ref: 'Property',
             isRequired: true,
             knexOptions: { isNotNullable: true }, // Relationship only!
-            kmigratorOptions: { null: false, on_delete: 'models.PROTECT' },
+            // If property is deleted => delete all tickets connected with this property
+            kmigratorOptions: { null: false, on_delete: 'models.CASCADE' },
```

Comment:
Maybe add related test case for that?

Reply:
No, since CASCADE deletion will be added later